cargo-workspaces/src/init.rs

```diff
@@ -1,18 +1,42 @@
 use crate::utils::{info, Error, Result};
+use clap::ArgEnum;
 
 use cargo_metadata::MetadataCommand;
+use cargo_util_schemas::manifest::{TomlManifest, TomlWorkspace};
 use clap::Parser;
 use dunce::canonicalize;
 use glob::glob;
 
 use std::{collections::HashSet, fs::write, path::PathBuf};
 
+#[derive(Debug, Clone, Copy, ArgEnum, Default)]
+enum Resolver {
+    #[clap(name = "1")]
+    V1,
+    #[default]
+    #[clap(name = "2")]
+    V2,
+}
```

Comment:
Doesn't everything default to resolve v2 nowadays? I don't see a use for this.

Reply:
I just wanted to be conservative as this crate also offers the ability to use old rust editions like e.g. 2015. I'd happily remove the v1 resolver stuff if you want